installer/packer/scripts/kov/configure_kov.sh

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/bash
```

Comment:
just confirmed with Fabio. he said that's to avoid dealing with expirations.
but producing an env file may be a good idea if we don't mind making those variables accessible to the whole world.